Makefile

```diff
@@ -384,11 +384,14 @@ agbot-package: agbot-image
 	fi
 
 anax-k8s-image: anax-k8s-clean
-	cp $(EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR)
-	cp $(CLI_EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR)
-	cp -f $(LICENSE_FILE) $(ANAX_K8S_CONTAINER_DIR)
-	@echo "Producing ANAX K8S docker image $(ANAX_K8S_IMAGE_STG)"
-	cd $(ANAX_K8S_CONTAINER_DIR) && docker build $(DOCKER_MAYBE_CACHE) $(ANAX_K8S_IMAGE_LABELS) -t $(ANAX_K8S_IMAGE_STG) -f Dockerfile.ubi . && \
+	cp $(EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR);
+	cp $(CLI_EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR);
+	cp -f $(LICENSE_FILE) $(ANAX_K8S_CONTAINER_DIR);
+	@echo "Producing ANAX K8S docker image $(ANAX_K8S_IMAGE_STG)";
+	if [[ $(arch) == "amd64" || $(arch) == "ppc64el" ]]; then \
+		echo "Using image for $(arch) arch"; \
+		cd $(ANAX_K8S_CONTAINER_DIR) && docker build $(DOCKER_MAYBE_CACHE) $(ANAX_K8S_IMAGE_LABELS) -t $(ANAX_K8S_IMAGE_STG) -f Dockerfile.ubi.$(arch) .; \
+	fi
```

Comment:
Is this just adding a debug echo? The echo above this already prints `$(ANAX_K8S_IMAGE_STG)`, which contains `$(arch)`, so why isn't that sufficient?

Reply:
Thx, will remove it

Reply:
fixed